### DISCUSSION TITLE
Password recovery

### DISCUSSION THREAD

**Discussion by SP6VWX** [signomix/signomix-main#54] [Q&A] [answered]
There is a button on a Login page to recover old password. After entering required data, system send an e-mail.
Until this point all is fine.
Clicking on received link, browser opens page with Signomix logo and Home Page link. Nothing more.
Tested on Firefox, Safari and Chrome.
Must miss some tricks, so tips welcome.

**Comment by gskorupa** [comment, marked as answer]
It looks like the password recovery mechanism has been broke by recent changes. I have to fix this.

Send me information to signomix@experiot.pl from the e-mail address provided when creating the account, then I will send you a temporary password.
